examples/therapy/therapy/app.py

```diff
@@ -33,6 +35,9 @@ class TherapyApp(toga.App):
             subtitle=response,
         )
 
+        # Scroll so the most recent entry is visible.
+        self.chat.scroll_to_row(len(self.chat.data) - 1)
```

Comment:
Now that I see this, should not we have some convenience method like

`scroll_to_last_row()`, `scroll_to_first_row()` or
`scroll_to_start()`, `scroll_to_end()`?

?

Reply:
Makes sense - I'll add that when I get a chance.